internal/wasm/store.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/tetratelabs/wazero/sys"
 )
 
+const initialNameToNodeListSize = 10_000
```

Comment:
I don't know the right number here either, I guess this is the size of the total in-flight modules? Whatever this controls we should add as a comment.

Reply:
Yeah ideally it would be some kind of average of the expected in-flight modules. Really some number that is low enough not to consume too much memory but high enough to fit a typical set of named modules. Since it can still grow I think undershooting is better than overshooting

Reply:
gotcha, and I think it is cute that you feel 10k is a low number of in-flight modules. seems you may be in a competition with @ShivanshVij on wasm throughput? ;)

Reply:
lol, in truth it's more that I thought 10k string headers was not much memory which is still telling…

Reply:
The following only matters if the variable is tied to performance/throughput:

Could we make this number editable? Like even if it was an exported variable as long as it was a knob we could tweak I’d be happy.

Reply:
Yes, I think it can be made configurable!

Reply:
Would we have to move the variable out of the internal package then?

Reply:
I’d be just as happy if we just made it possible to disable (and keep it enabled by default, that doesn’t matter to us)